tests/unit_tests/data/omni_modal/test_loaders.py

```
@@ -43,7 +43,7 @@ def test_build_mimo_data_loaders_raises_when_model_not_mimo(monkeypatch):
     cfg = SimpleNamespace(model=object(), train=SimpleNamespace(micro_batch_size=2))
     provider = FakeProvider()
 
-    with pytest.raises(ValueError, match="cfg.model must be MimoModelProvider"):
+    with pytest.raises(ValueError, match="cfg.model must be OmniModalProvider"):
```

Ruff (0.15.10)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)